.github/workflows/build.yml

```diff
@@ -161,34 +161,6 @@ jobs:
         run: cd .repo && npx projen package:python
```

Comment:
Run `npx projen` locally (don't need to run the entire build process) and commit the change so that these build files return to normal